Source/Blazorise.Bootstrap5/Bootstrap5ClassProvider.cs

```diff
@@ -657,21 +657,21 @@ public override string Container( Breakpoint breakpoint )
 
     public override string Bar() => "navbar";
 
-    public override string BarInitial( bool initial ) => initial ? "b-bar-initial" : null;
+    public override string BarInitial( bool initial ) => initial ? "navbar-initial" : null;
 
     public override string BarAlignment( Alignment alignment ) => FlexAlignment( alignment );
 
-    public override string BarThemeContrast( ThemeContrast themeContrast ) => $"navbar-{ToThemeContrast( themeContrast )} b-bar-{ToThemeContrast( themeContrast )}";
+    public override string BarThemeContrast( ThemeContrast themeContrast ) => $"navbar-{ToThemeContrast( themeContrast )}";
 
     public override string BarBreakpoint( Breakpoint breakpoint ) => breakpoint != Breakpoint.None && breakpoint != Breakpoint.Mobile ? $"navbar-expand-{ToBreakpoint( breakpoint )}" : null;
 
-    public override string BarMode( BarMode mode ) => $"b-bar-{ToBarMode( mode )}";
+    public override string BarMode( BarMode mode ) => mode != Blazorise.BarMode.Horizontal ? $"navbar-{ToBarMode( mode )} flex-column" : null;
```

Comment:
Not an issue, but I think it reads better in the **positive** `mode == Blazorise.BarMode.Horizontal` and invert acorrdingly.

Also that's how It is done in other places below, so I'd just keep it consistent
![image](https://github.com/Megabit/Blazorise/assets/22283161/f9dd60a2-a3b4-4cd8-8d5a-22c116ec1e78)